client/cypress/e2e/downloadPacket.spec.js

```diff
@@ -2,20 +2,22 @@
 
 describe('Census Block Group download', () => {
   it('validate file download', () => {
-    const filename = `usa.zip`;
+    // const filename = `usa.zip`;
     cy.visit('localhost:8000/en/cejst');
-    cy.get('#download-link').invoke('attr', 'target', '_blank');
-    cy.intercept(`https://justice40-data.s3.amazonaws.com/Score/${filename}`,
-        {
-          body: 'success',
-          headers: {
-            'Content-Type': 'text/html; charset=utf-8',
-            'Content-Disposition': 'attachment',
-          },
-        },
-    ).as('downloadRequest');
-    cy.get('button[class*="downloadPacket"]').click();
-    cy.wait('@downloadRequest');
-    cy.readFile(`cypress/downloads/${filename}`).should('exist');
+
+    // Todo VS: Download packet component is being moved. Will be re-enabled with
+    // cy.get('#download-link').invoke('attr', 'target', '_blank');
+    // cy.intercept(`https://justice40-data.s3.amazonaws.com/Score/${filename}`,
+    //     {
+    //       body: 'success',
+    //       headers: {
+    //         'Content-Type': 'text/html; charset=utf-8',
+    //         'Content-Disposition': 'attachment',
+    //       },
+    //     },
+    // ).as('downloadRequest');
+    // cy.get('button[class*="downloadPacket"]').click();
+    // cy.wait('@downloadRequest');
+    // cy.readFile(`cypress/downloads/${filename}`).should('exist');
```

Comment:
Now that we have https://justice40-data.s3.amazonaws.com/data-pipeline/data/score/downloadable/Screening+Tool+Data.zip , can we add back this test?

Reply:
Not yet. The download packet component was removed and will be placed on the methodology page. There is a ticket for that. Once that ticket is started, this will code will be reinstated.